conformance/utils/suite/suite.go

```diff
@@ -222,7 +243,7 @@ func (test *ConformanceTest) Run(t *testing.T, suite *ConformanceTestSuite) {
 	// Check that all features exercised by the test have been opted into by
 	// the suite.
 	for _, feature := range test.Features {
-		if supported, ok := suite.SupportedFeatures[feature]; !ok || !supported {
+		if suite.SupportedFeatures.Has(feature) {
```

Comment:
I think this we actually want this?
```suggestion
		if !suite.SupportedFeatures.Has(feature) {
```

Reply:
@robscott It's my mistake, updated.